modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -38,14 +38,14 @@ definition {
 	@description = "This is a test for LPS-171642. User can access MDF Request Listing through Quick Links"
 	@priority = "4"
 	test CanAccessMDFRequestListingViaQuickLinks {
-		task ("When I click on the MDF") {
+		task ("When I click on 'MDF' button, on Quick Links section") {
 			Click(locator1 = "PRMHomePage#CLICK_MDF");
 		}
 
-		task ("Then it was directed to MDF Requests") {
+		task ("Then I'm redirected to the 'MDF Request Listing' page") {
 			AssertElementPresent(
 				key_title = "MDF Requests",
-				locator1 = "PRMHomePage#ASSERT_REQUESTS");
+				locator1 = "Header#H1_TITLE");
```

Comment:
Hey @juliosilveira77, here you can use an existing macro. `PRMPages.assertPageTitle`.